package.json

```diff
@@ -42,6 +42,7 @@
   "license": "MIT",
   "dependencies": {
     "electron": "^1.4.12",
+    "electron-default-menu": "^1.0.0",
```

Comment:
Might need to be merged with a recent change to the package.json to clamp the version numbers (this helps predictability of the build, so that newer versions of libraries don't break us). Can you please merge that and then change the version here to "1.0.0" instead of "^1.0.0"?